servers/audio/effects/audio_effect_limiter.cpp

```diff
@@ -120,15 +147,15 @@ void AudioEffectLimiter::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_soft_clip_ratio", "soft_clip"), &AudioEffectLimiter::set_soft_clip_ratio);
 	ClassDB::bind_method(D_METHOD("get_soft_clip_ratio"), &AudioEffectLimiter::get_soft_clip_ratio);
 
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "ceiling_db", PROPERTY_HINT_RANGE, "-20,-0.1,0.1"), "set_ceiling_db", "get_ceiling_db");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "threshold_db", PROPERTY_HINT_RANGE, "-30,0,0.1"), "set_threshold_db", "get_threshold_db");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "ceiling_db", PROPERTY_HINT_RANGE, "-20,0,0.1"), "set_ceiling_db", "get_ceiling_db");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "threshold_db", PROPERTY_HINT_RANGE, "-30,-0.1,0.1"), "set_threshold_db", "get_threshold_db");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "soft_clip_db", PROPERTY_HINT_RANGE, "0,6,0.1"), "set_soft_clip_db", "get_soft_clip_db");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "soft_clip_ratio", PROPERTY_HINT_RANGE, "3,20,0.1"), "set_soft_clip_ratio", "get_soft_clip_ratio");
 }
 
 AudioEffectLimiter::AudioEffectLimiter() {
-	threshold = 0;
-	ceiling = -0.1;
+	threshold = -0.1;
```

Comment:
Here and above (in the ADD_PROPERTY lines): is this just a bug that you're correcting? Threshold should always be quieter than the absolute limit for a limiter, so this sounds right to me but I want to confirm.

I also want to make 100% sure that you know that these property hints can be bypassed from script. So even if `ceiling < threshold` somehow we should still avoid crash (which I think we do).

Reply:
It doesn't make sense for the threshold to be above the ceiling. Would it be appropriate to add a check in the setter and getter of Threshold and Ceiling to ensure that threshold never exceeds the ceiling? Something like:

`if (threshold > ceiling) {
	threshold = ceiling;
}`

This is what I've implemented for now, and I added the side-effect to the documentation. I can change that before I update my PR with a new commit though.